inductiva/resources/machines.py

```diff
@@ -71,11 +83,12 @@ def estimate_cloud_cost(self):
         specified configurations up in the cloud. The actual cost may vary.
 
         Returns:
-            The estimated cost per hour of the machine group in US
+            The estimated cost per hour of the machine group, in US
               dollars ($/h)."""
         #TODO: Contemplate disk size in the price.
-        estimated_cost = super()._get_estimated_cost() * self.num_machines
-        logging.info("Estimated cloud cost per hour for all machines : %s $/h",
+        estimated_cost = super()._get_estimated_cost(
+            self.spot) * self.num_machines
```

Comment:
the breaking of lines make it difficult to follow.

```suggestion
       cost_per_machine = super()._get_estimated_cost(self.spot)
       estimated_cost =  cost_per_machine * self.num_machines
```